packages/trader/src/AppV2/Components/EntryExitDetails/entry-exit-details.tsx

```diff
@@ -0,0 +1,89 @@
+import { Text, CaptionText } from '@deriv-com/quill-ui';
+import CardWrapper from '../CardWrapper';
+import React, { useMemo } from 'react';
+import { TContractInfo, addComma, getEndTime } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const getDateTimeFromEpoch = (epoch?: number) => {
+    if (epoch) {
+        const date = new Date(epoch * 1000);
+        const formattedDate = date.toUTCString().split(' ').slice(0, 4).join(' ');
+        const formattedTime = `${date.toUTCString().split(' ')[4]} GMT`;
+        return {
+            date: formattedDate,
+            time: formattedTime,
+        };
+    }
+};
+
+const DateTimeRow = ({
+    label,
+    value,
+    date,
+    time,
+}: {
+    label: React.ReactNode;
+    value?: string;
+    date: string;
+    time: string;
+}) => (
+    <div className='entry-exit-details__table-row'>
+        <div className='entry-exit-details__table-cell'>
+            <Text size='sm' color='quill-typography__color--subtle'>
+                {label}
+            </Text>
+        </div>
+        <div className='entry-exit-details__table-cell'>
+            <Text size='sm'>{value}</Text>
```

Comment:
`value` is optional according to type. Maybe we need to add conditional rendering? Same for `label`